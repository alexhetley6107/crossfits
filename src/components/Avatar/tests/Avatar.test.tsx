// import { render, screen } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
// import { Avatar } from '../Avatar';
import '@testing-library/jest-dom';
import 'jest-styled-components';

describe('Avatar component', () => {
 describe('General', () => {
    it('Render', () => {
      // render(<Avatar />);
      // const target = screen.getByText('Avatar');
      // expect(target).toBeInTheDocument();
      expect('Render').toBe('Render');
    });
  });
});
