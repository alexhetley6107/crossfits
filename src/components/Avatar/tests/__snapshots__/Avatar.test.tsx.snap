// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Avatar component > Props > Text 1`] = `
.c0 {
  height: 32px;
  width: 32px;
  border-radius: 1000px;
  border: 1px solid #808080;
  display: flex;
  justify-content: center;
  align-items: center;
  overflow: hidden;
}

.c1 {
  color: #333;
}

<div
  class="c0"
  data-testid="Avatar"
>
  <span
    class="c1"
  >
    AB
  </span>
</div>
`;
